src/components/dropdown-group/dropdown-group.tsx

```diff
@@ -36,17 +36,19 @@ export class DropdownGroup {
   //
   //--------------------------------------------------------------------------
 
-  /** optionally set a group title for display */
-  @Prop({ reflect: true }) groupTitle?: string;
+  /** Specifies and displays a group title. */
+  @Prop({ reflect: true }) groupTitle: string;
 
   /**
-    specify the selection mode - multiple (allow any number of (or no) active items), single (allow and require one active item),
-   none (no active items), defaults to single
+   * Specifies the component's selection mode, where
+   * `"multi"` allows any number of (or no) selected `calcite-dropdown-item`s,
```

Comment:
Do we need these in quotes?  `"multi"` vs `multi`?

Reply:
For some reason we didn't document it in the original issue, but its to discern string values in the docs similar to this [Popover `overlayPositioning` prop](https://developers.arcgis.com/calcite-design-system/components/popover/#component-api-properties-overlayPositioning).

Reply:
Ok sounds good. I just thought the strings were maybe unnecessary.